oauth-common/src/main/java/io/strimzi/kafka/oauth/common/LogUtil.java

```diff
@@ -35,4 +35,20 @@ public static String mask(String input) {
 
         return input.substring(0, 4) + "**" + input.substring(len - 4, len);
     }
+
+    /**
+     * Collect all exception causes into a single string
+     *
+     * @param e The exception
+     * @return The message with all the causes
+     */
+    public static String getCauseMessage(Throwable e) {
```

Comment:
Should it be maybe named in plural? `getCauseMessages`? Or maybe even `getAllCauseMessages` to make this a bit more self-explaining?

Reply:
Makes sense.